src/interfaces/IPolygonMigration.sol

```diff
@@ -11,8 +11,9 @@ import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 interface IPolygonMigration {
     /// @notice emitted when MATIC are migrated to POL
     /// @param account the account that migrated MATIC
+    /// @param recipient the account that received POL
     /// @param amount the amount of MATIC that was migrated
-    event Migrated(address indexed account, uint256 amount);
+    event Migrated(address indexed account, address recipient, uint256 amount);
```

Comment:
Hi, thanks. I'd suggest we confirm if this event signature "change" is breaking any known relevant event listeners to agree on the best way to mitigate its impact (i.e. sometimes new events were created).